.gitmodules

```diff
@@ -0,0 +1,16 @@
+[submodule "containers/docker.php7"]
+	path = containers/docker.php7
+	url = git@github.com:drud/docker.php7.git
+	branch = 20170411_php7_experiment
```

Comment:
The nightly_build.mak actually gets the upstream/master in all cases. But I'll check it. I actually put that in there to test the behavior I think, and it snuck in.